### PR TITLE
Pixel perfect

### PR DESCRIPTION
**Pull request by AvnerGuyAvrahamCom**

### CODE REVIEW

File: src/Components/CheckoutModal/CheckoutModalBody.tsx [hunk 1]
```diff
@@ -1,23 +1,47 @@
...
-        <Route
-          key={route.path}
-          path={route.path}
-          element={<route.component />}
-        />
-      ))}
-    </Routes>
-  </>
-);
+const BoxContainerStyle = (isMobile: boolean) => {
```
Comment (ValentineGuyAvrahamCom) [inline src/Components/CheckoutModal/CheckoutModalBody.tsx:15]: fix like we do before I send u an example

File: src/Components/CheckoutModal/OrderSummary/GeneralDetails.tsx [hunk 2]
```diff
@@ -15,19 +15,41 @@ export const GeneralDetail: FC<IGeneralDetail> = ({
...
+      justify="space-between"
+      w="full"
+      fontWeight={isBold ? 'bold' : 'normal'}
+      my="1"
+    >
+      <Text fontSize="small" textColor={isGray ? 'GrayText' : 'blackAlpha.900'}>
+        {name}
+      </Text>
+      <Text fontSize="small" textColor={isGray ? 'GrayText' : 'blackAlpha.900'}>
+        {currencyFigure(currency, value)}
```
Comment (ValentineGuyAvrahamCom) [inline src/Components/CheckoutModal/OrderSummary/GeneralDetails.tsx:50]: I think u can replace it with `{value !== 0 && currency}`

File: src/Components/CheckoutModal/Sections/Shipping/FormInput.tsx [hunk 3]
```diff
@@ -20,7 +20,7 @@ interface IFormInput {
   name: string;
   label: string;
   type: string;
-  w: string | undefined;
+  w?: string | undefined;
```
Comment (ValentineGuyAvrahamCom) [inline src/Components/CheckoutModal/Sections/Shipping/FormInput.tsx:23]: if u have `?:` u dont need this `| undefined`

File: src/Components/CheckoutModal/Sections/Shipping/index.tsx [hunk 4]
```diff
@@ -43,18 +50,65 @@ export const Shipping = () => {
...
+      {!isOpen && (
+        <Button
+          my="3"
+          onClick={handleClick}
+          variant="summary"
+          w="full"
+          leftIcon={<MdAdd />}
+          justifyContent="flex-start"
+        >
+          <Text fontSize="11px">APARTMENT, BUILDING, FLOOR</Text>
```
Comment (ValentineGuyAvrahamCom) [inline src/Components/CheckoutModal/Sections/Shipping/index.tsx:62]: move this text to messages file

File: src/Components/CheckoutModal/OrderSummary/GeneralDetails.tsx (hunk 2 above)
Comment (AvnerGuyAvrahamCom) [reply src/Components/CheckoutModal/OrderSummary/GeneralDetails.tsx:50]: done

File: src/Components/CheckoutModal/CheckoutModalBody.tsx (hunk 1 above)
Comment (AvnerGuyAvrahamCom) [reply src/Components/CheckoutModal/CheckoutModalBody.tsx:15]: done

File: src/Components/CheckoutModal/OrderSummary/index.tsx [hunk 5]
```diff
@@ -9,22 +9,24 @@ import { GeneralDetail } from './GeneralDetails';
...
 
 const ContainerStyles: BoxProps = {
   pt: '6',
   px: '8',
-  h: 'full',
-  w: { base: 'full', md: '40%' },
+  h: 'auto',
+  w: { base: 'full', md: '25rem' },
   bg: 'summaryBG',
   borderRightRadius: { md: 'md', base: undefined },
```
[the remark this answers is not in the document]
Comment (AvnerGuyAvrahamCom) [reply src/Components/CheckoutModal/OrderSummary/index.tsx:22]: done

File: src/Components/CheckoutModal/Sections/Shipping/FormInput.tsx (hunk 3 above)
Comment (AvnerGuyAvrahamCom) [reply src/Components/CheckoutModal/Sections/Shipping/FormInput.tsx:23]: done

File: src/Components/CheckoutModal/Sections/Shipping/index.tsx (hunk 4 above)
Comment (AvnerGuyAvrahamCom) [reply src/Components/CheckoutModal/Sections/Shipping/index.tsx:62]: done

**Review by ValentineGuyAvrahamCom — APPROVED** [review]

File: src/Components/CheckoutModal/ProgressBar/ControlButton.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+  Flex,
+  FlexProps,
+  Icon,
+} from '@chakra-ui/react';
+import { FC } from 'react';
+import { IconType } from 'react-icons';
+
+interface IControlsButton {
+  text: string;
+  func: () => void;
```
Comment (ValentineGuyAvrahamCom) [inline src/Components/CheckoutModal/ProgressBar/ControlButton.tsx:14]: please give correct name of function

File: src/Components/CheckoutModal/ProgressBar/ControlButton.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  icon: IconType;
+}
+
+const ButtonStyle: ButtonProps = {
+  fontSize: '11',
+  pt: '1',
+  pb: '3',
+  h: '8',
+  w: '8',
+  variant: 'unstyled',
```
Comment (ValentineGuyAvrahamCom) [inline src/Components/CheckoutModal/ProgressBar/ControlButton.tsx:24]: variant its not Button Style move it to button prop

**Review by ValentineGuyAvrahamCom — CHANGES_REQUESTED** [review]

File: src/Components/CheckoutModal/ProgressBar/ControlButton.tsx (hunk 7 above)
Comment (AvnerGuyAvrahamCom) [reply src/Components/CheckoutModal/ProgressBar/ControlButton.tsx:24]: done

File: src/Components/CheckoutModal/ProgressBar/ControlButton.tsx (hunk 6 above)
Comment (AvnerGuyAvrahamCom) [reply src/Components/CheckoutModal/ProgressBar/ControlButton.tsx]: done

**Review by ValentineGuyAvrahamCom — APPROVED** [review]

**Pull request merged**